lib/core/entities/artist_entity/artist_entity.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:flutter/foundation.dart';
+import '../music_entity/music_entity.dart';
+
+part 'artist_entity.freezed.dart';
+
+@freezed
+class Artist with _$Artist {
+  factory Artist({
+    required String id,
+    required String name,
+    required String about,
+    required List<String?> socialNetwork,
+    required List<String> photos,
+    required List<Music> music,
+  }) = _Artist;
+}
```

Comment:
missed the fromJson just like the JudgeEntity

Reply:
Fixed.